lib/update-lockfile.js

```diff
@@ -32,7 +32,9 @@ module.exports = function updateLockfile (dependency, options) {
 
     if (options.yarn) {
       exec(`yarn add ${args}`)
-    } else {
+    }
+
+    if (options.npm) {
```

Comment:
This would break existing implementations of the default behavior of running npm install if there was no options passed. The change should probably be something like

```js
if (!options || options.npm) {
...
}